Eatery Blue/UI/EateryCard/EateryLargeCardContentView.swift

```diff
@@ -196,11 +198,15 @@ class EateryLargeCardContentView: UIView {
         }
 
         favoriteButton.snp.makeConstraints { make in
-            make.trailing.equalToSuperview().inset(12)
-            make.top.equalTo(imageView.snp.bottom).offset(12)
-            make.leading.equalTo(labelStackView.snp.trailing).offset(16)
-            make.width.equalTo(20)
-            make.height.equalTo(20)
+            make.trailing.equalToSuperview()
+            make.top.equalTo(imageView.snp.bottom)
+            make.leading.equalTo(labelStackView.snp.trailing).offset(4)
+            make.size.equalTo(44)
+        }
```

Comment:
Can you test this with different screen sizes?

Reply:
looks good on iphone se 3rd generation, iphone 15 pro, iphone 15 pro max, and ipad pro 4th generation 11 inch